test/regress/regress1/ho/bug_freevar_PHI004^4-delta.smt2

```diff
@@ -1,4 +1,4 @@
-; COMMAND-LINE: --finite-model-find -q --decision=justification-old
+; COMMAND-LINE: --finite-model-find --decision=justification-old -q --ho-elim
```

Comment:
Why add --ho-elim? If there is something here to be tested then you could have the two cases:
```
; COMMAND-LINE: --finite-model-find --decision=justification-old -q
; COMMAND-LINE: --finite-model-find --decision=justification-old -q --ho-elim
```

Reply:
I was trying to patch a fix to this regression (it is disabled by this PR), I'll revert this change.